src/SimpleGraphs/simpledigraph.jl

```diff
@@ -566,3 +566,32 @@ function rem_vertices!(g::SimpleDiGraph{T},
 
     return reverse_vmap
 end
+
+function all_neighbors(g::SimpleDiGraph{T}, u::T) where T
+   common = Vector{T}()
```

Comment:
I named the variable `common` in a hurry, could you change it to something more suitable ? Like `union_in_out` or something else ?